docs/README.md

```diff
@@ -459,7 +495,163 @@ Steps:
+      name: default
+    spec:
+      targetRef:
+        kind: Istio
+        name: default
+    EOF
+    ```
+
+3. Confirm the control plane is installed and is using the desired version.
+
+    ```console
+    $ kubectl get istio
+    NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+    default   1           1       1        default-v1-22-5   Healthy   v1.22.5   52s
+    ```
+    Note: `IN USE` field shows as 1, even though no workloads are using the control plane. This is because the `IstioRevisionTag` is referencing it.
+
+4. Inspect the `IstioRevisionTag`.
+
+    ```console
+    $ kubectl get istiorevisiontags
+    NAME      STATUS                    IN USE   REVISION          AGE
+    default   NotReferencedByAnything   False    default-v1-22-5   52s
+    ```
+
+5. Create `bookinfo` namespace and label it to mark it for injection.
+
+    ```bash
+    kubectl create namespace bookinfo
+    kubectl label namespace bookinfo istio-injection=enabled
+    ```
+
+6. Deploy bookinfo application.
+
+    ```bash
+    kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.22/samples/bookinfo/platform/kube/bookinfo.yaml
+    ```
+
+7. Review the `IstioRevisionTag` resource after application deployment.
+
+    ```console
+    $ kubectl get istiorevisiontag
+    NAME      STATUS    IN USE   REVISION          AGE
+    default   Healthy   True     default-v1-22-5   2m46s
+    ```
+    Note: `IN USE` field shows 'True', as the tag is now referenced by both active workloads and the bookinfo namespace.
+
+8. Confirm that the proxy version matches the control plane version.
+
+    ```bash
+    istioctl proxy-status 
+    ```
+    The column `VERSION` should match the control plane version.
+
+9. Update the control plane to a new version.
+
+    ```bash
+    kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.23.2"}}'
+    ```
+
+10. Verify the `Istio`, `IstioRevision` and `IstioRevisionTag` resources. There will be a new revision created with the new version.
+
+    ```console
+    $ kubectl get istio
+    NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+    default   2           2       1        default-v1-23-2   Healthy   v1.23.2   9m23s
+
+    $ kubectl get istiorevision
+    NAME              TYPE    READY   STATUS    IN USE   VERSION   AGE
+    default-v1-22-5   Local   True    Healthy   True     v1.22.5   10m
+    default-v1-23-2   Local   True    Healthy   True    v1.23.2   66s
+
+    $ kubectl get istiorevisiontag
+    NAME      STATUS    IN USE   REVISION          AGE
+    default   Healthy   True     default-v1-23-2   10m44s
+    ```
+    Now, both our IstioRevisions and the IstioRevisionTag are considered in use. The old revision default-v1-22-5 because it is being used by proxies, the new revision default-v1-23-2 because it is referenced by the tag, and lastly the tag because it is referenced by the bookinfo namespace.
```

Comment:
```suggestion
    Now, both our IstioRevisions and the IstioRevisionTag are considered in use. The old revision default-v1-23-3 because it is being used by proxies, the new revision default-v1-24-1 because it is referenced by the tag, and lastly the tag because it is referenced by the bookinfo namespace.
```